src/lib.rs

```diff
@@ -1229,6 +1252,7 @@ impl<T: Counter> Histogram<T> {
     ///
     /// Note that the return value is capped at `u64::max_value()`.
     pub fn median_equivalent(&self, value: u64) -> u64 {
+        // TODO isn't this just saturating?
```

Comment:
Yup. I guess I just didn't know about `saturating_add` at the time. Happy to have it changed.